src/views/components/trace/trace-chart-table/trace-item.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+  <div>
+      <div @click="showSelectSpan" :class="['trace-item', 'level'+( data.level - 1)]">
+        <div :class="['method', 'level'+( data.level - 1)]">
+          <span 
+              v-if="data.children && data.children.length > 0" 
+              @click.stop="toggle" 
+              :class="['trace-table-toggle', displayChildren? 'collapse': 'expand']">
+
+          </span>
+          <span v-tooltip:bottom.inTraceTable="data.endpointName">
+            {{data.endpointName}}
+          </span>
+        </div>
+        <div class="start-time">
+          {{formatTime(data.startTime)}}
+        </div>
+        <div class="gap">
+          0
+        </div>
+        <div class="exec-ms">
+         {{(data.endTime - data.startTime)?(data.endTime - data.startTime) : '0'}} 
+        </div>
+        <div class="exec-percent">
+          <div class="outer-progress_bar" :style="{width: outterPercent}">
+              <div class="inner-progress_bar" :style="{width: innerPercent}"></div>
+          </div>
+        </div>
+        <div class="self">
+          {{data.dur ? data.dur + '' : '0'}}
+        </div>
+        <div class="api">
+          <span v-tooltip:bottom="data.component||'-'">{{data.component || '-'}}</span>
+        </div>
+        <div class="application">
+          <span v-tooltip:bottom="data.serviceCode||'-'">{{data.serviceCode}}</span>
+        </div>
+      </div>      
+    <div v-show="data.children && data.children.length > 0 && displayChildren" class="children-trace">
+        <item v-for="(item, index) in data.children" :key="index" :data="item"> </item>
+    </div>
+  </div>
+
+</template>
+<style lang="scss" scoped>
+  @for $i from 0 through 20 {
+    .method.level#{$i} {
+      text-indent: #{$i * 10}px;
```

Comment:
Only 20-tier structure is supported here. Use DOM recursive nesting to achieve infinite tier